test/generator/exercise_case/case_helpers_test.rb

```diff
@@ -57,6 +57,29 @@ def test_heredoc
         expected = "<<~TEXT\n foo\n bar\nTEXT"
         assert_equal expected, HeredocCase.new.workload
       end
+
+      class ToStringCase
+        include CaseHelpers
+
+        def workload(phrases)
+          to_string(phrases)
+        end
+      end
+      def test_to_string_with_two_strings
```

Comment:
NIce! And could you please add whitespace between the tests?
(Unless there's a secret convention that I'm not aware of for following the spacing as it is in line 56. )